web/html/src/manager/state/highstate-summary.tsx

```diff
@@ -53,6 +53,7 @@ export default function HighstateSummary({ minionId }) {
   }
 
   return (
+    // TODO: This identifier seems wrong, what's the correct identifier here?
```

Comment:
Since I'm not really familiar with the logic here, I'm not sure what the identifier should be in this context. The items do not have a `state` property as far as I could see.

Reply:
Me neither. That's code that @cbbayburt recently added. He should be able to answer it.

Reply:
I created https://github.com/SUSE/spacewalk/issues/15856 to track this so we can go ahead with this merger.